apply-roles

```diff
@@ -170,9 +218,9 @@ apply_roles_to_subscription() {
   local is_custom_role_assigned
   local is_reader_role_assigned
 
-  echo "- Subscription: ${subscription_id}"
+  echo " - Subscription: ${subscription_id}"
 
-  echo "  - Checking custom role assignment..."
+  echo " - Checking custom role assignment..."
```

Comment:
⛏️ The spacing before the `-` was intentional to have a nice output on the command line by showing nested/related commands. By removing this spacing this formatting has been lost. Please review the output and decide if it is ok to have everything at the same level?

Reply:
I have modified it so it is nested, would you run it a couple times and see if it aligns with your expectations?